### DISCUSSION TITLE
Popper RightAligned

### DISCUSSION THREAD

**Discussion by David-Moreira** [floating-ui/floating-ui#1611] [Q&A] [answered]
Hello,

Any way I can have my popper to right align to the reference element? 
Just like this:
![image](https://user-images.githubusercontent.com/22283161/162574009-45146185-3815-4a57-ba0e-3c2f35f57787.png)

**Comment by FezVrasta** [comment, marked as answer]
You can use the -start and -end placement variations to achieve this

**Reply by David-Moreira** [reply]
Hello,
The end placement will actually place the menu starting from the end of the reference element.

**Reply by David-Moreira** [reply]
I am sorry I just re-checked your demo and it does seem that it does what I want. 
![image](https://user-images.githubusercontent.com/22283161/162574760-56ed4887-a4f5-4b1e-b421-abe5913df5c1.png)


Maybe it's something wrong with our implementation that ends up looking like this:
![image](https://user-images.githubusercontent.com/22283161/162574779-2e89287d-d9a8-4703-9e6e-1f410e3c5902.png)

, I'll take the time to troubleshoot more.

**Reply by atomiks** [reply]
@David-Moreira I think that was a bug in an earlier version of Popper when it was in an `overflow: hidden` container, but it should be fixed in the latest version
